### ISSUE TITLE
Demands the removal of the Korean movie The New Normal.

### ISSUE THREAD

**Issue by UNPA-STUDIOS** [btdig/dhtcrawler2#31]
We are UNPA STUDIOS, a Korean movie production company.
We demand that you remove the movie NEW NORMAL, which we produced and released in 2023, from your site. 
The torrent distribution of this movie violates copyright protection laws.
Please remove it as soon as possible.

![new normal 2024-07-23 pm 12 28 17](https://github.com/user-attachments/assets/7661fa74-c393-429f-9c6c-3b387a2fc51f)
![new normal 2024-07-23 pm 12 27 44](https://github.com/user-attachments/assets/99458581-8230-4330-9978-88ea3e7167a0)

**Comment by milahu** [comment]
thanks for your recommendation

i will download and watch "your" movie
and if its somewhat interesting
i will permaseed the shit out of it with my 500Mbit uplink : )

for future recommendations:
please keep your stupid screenshots and just send magnet links

```
magnet:?xt=urn:btih:2b8c525e88b2b14510f7650538e03f94f251f37c&dn=New.Normal.2022.720p.WEBRip.x264.AAC-[YTS.MX]&tr=udp://185.243.218.213:80/announce

magnet:?xt=urn:btih:05be53fe57058a0d02a5b141986bb2cfbcaf155b&dn=New.Normal.2022.1080p.WEBRip.x264.AAC5.1-[YTS.MX]&tr=udp://185.243.218.213:80/announce

magnet:?xt=urn:btih:b7961ceb33670e91223f3e530e5f395cd81eaf3b&dn=New.Normal.2022.1080p.Korean.WEB-DL.HEVC.x265.5.1.BONE.mkv&tr=udp://185.243.218.213:80/announce
```

**Comment by milahu** [comment]
> if its somewhat interesting

nah. just a bunch of koreans killing each other. boring. i want my money back!
